.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "printWidth": 120
```

Comment:
MB add trailing comma ?)

Reply:
What is this question about?

Reply:
add comma after ```120```
```120,```

Reply:
It is not allowed to use trailing comma in JSON format.